src/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/InstanceData.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Diagnostics
+{
+    /// <summary>
+    ///     A holder of instance data.
+    /// </summary>    
+    public class InstanceData
+    {
+        private readonly string _instanceName;
```

Comment:
remove these two.

Reply:
Why?

Reply:
see the next comment, that will explain it. You can turn it into a auto property which saves code.